GCRCatalogs/register.py

```diff
@@ -176,6 +186,28 @@ def set_root_dir_by_site(self, site):
             site_string = ' '.join(_config_register.site_list)
             warnings.warn(f"Unknown site '{site}'.\nAvailable sites are: {site_string}\nroot_dir is unchanged")
 
+    def persist_root_dir(self, value=None):
+        """
+        Write root_dir to user config. By default write current session value
+        Returns
+        -------
+        old value (may be None)
```

Comment:
Returning old value here is a somewhat unexpected behavior. Usually a function that sets something would either return nothing, or returns the new value that is being set. 

Is there a particular reason that this function must return the old value? If not, or if it can be achieved differently, I would suggest that this function either returns `None` or `value`.

Reply:
I think it might have been left over from the need to display the old value for script users.  Since that is now handled by a separate module, there is no need for routines inside register to do this.